src/examples/colors/generate_wcs_languages.py

```diff
@@ -131,33 +139,40 @@ def get_expression_from_language(color_name:str, language:ColorLanguage):
                 language_terms[lang_num] = []
             language_terms[lang_num].append(lang_term_abbrev)
 
+    # Generate the uniform probability prior as a baseline
+    # uniform_prior = np.array(color_universe._prior)
 
-    #Generate the uniform probability prior as a baseline
-    #uniform_prior = np.array(color_universe._prior)
-
-    #Munsell to Cielab hues
+    # Munsell to Cielab hues
     munsell_to_cielab = np.array(list(munsell_to_cielab.values()))
-    #print(f"Munsell to CIELAB hues:{munsell_to_cielab}")
+    # print(f"Munsell to CIELAB hues:{munsell_to_cielab}")
 
     SIGMA_SQUARED_SCALAR = 64
-    #Calculate the meaning space as an isotropic Gaussian centered at the first chip C, for all other points
-    def meaning_distance(center, point):
-        return math.exp((-1/(2*SIGMA_SQUARED_SCALAR) * np.linalg.norm(center-point)))
 
-    #Generate the meaning space
-    meaning_dists = np.zeros(shape=(len(color_universe.referents), len(color_universe.referents)))
+    # Calculate the meaning space as an isotropic Gaussian centered at the first chip C, for all other points
+    def meaning_distance(center, point):
+        return math.exp(
+            (-1 / (2 * SIGMA_SQUARED_SCALAR) * np.linalg.norm(center - point))
+        )
+
+    # Generate the meaning space
+    meaning_dists = np.zeros(
```

Comment:
Very helpful to see this is what `meaning_dists` is! Make clear in othe rmethods that use it that this is what it is

Reply:
I'll add a few more comments to this to make sure there's clarity.